content/applications/websites/ecommerce/managing_products/catalog.rst

```diff
@@ -0,0 +1,186 @@
+=======
+Catalog
+=======
+
+The eCommerce catalog is the equivalent of your physical store shelves: it allows customers to see
+what you have to offer. Clear categories, available options, sorting, and navigation threads help
+you structure it efficiently.
+
+Categorize the product catalog
+==============================
+
+In Odoo, there is a **specific category model** for your eCommerce. Using eCommerce categories for
+your products allows you to add a navigation menu on your eCommerce page. Visitors can then use it
+to view all products under the category they select. To do so, go to :menuselection:`Website app
+--> eCommerce --> Products`, select the product you wish to modify, click on the :guilabel:`Sales`
+tab. and select the :guilabel:`Categories` you want under :guilabel:`eCommerce Shop`.
+
+.. note::
+   A single product can appear under multiple eCommerce categories.
+
+When your product's categories are configured, go to :menuselection:`Website app --> Site -->
+Homepage --> Shop` and click on :menuselection:`Edit --> Customize tab`. In the
```

Comment:
```suggestion
When your product's categories are configured, go to :menuselection:`Website dashboard --> Shop` and click on :menuselection:`Edit --> Customize tab`. In the
```
My suggestion is to take the fastest path and skip 2 steps, but up to you to follow it, or not.
This logic could be applied to all similar paths.

Reply:
Noted